assets/js/autocomplete.ts

```diff
@@ -15,17 +15,21 @@ import {
   TermSuggestion,
 } from './utils/suggestions';
 
-type InputFieldElement = HTMLInputElement | HTMLTextAreaElement;
+type AcEnabledInputElement = HTMLInputElement | HTMLTextAreaElement;
```

Comment:
I don't really like the "Ac" abbreviation here. How about `AutocompletableInputElements`?